cypress/integration/host/host_events.ts

```diff
@@ -11,11 +13,10 @@ describe("Host events", () => {
 
   it("host events display the correct text", () => {
     cy.visit(pathWithEvents);
-    cy.dataCy("host-event-table-page-size-selector").click();
-    cy.dataCy("host-event-table-page-size-selector-100").click();
-
-    const dataCyTableRows = "[data-cy=host-events-table] tr td:first-child";
-    cy.get(dataCyTableRows).should("have.length.of.at.most", 32);
+    clickOnPageSizeBtnAndAssertURLandTableSize(
+      100,
+      "[data-cy=host-events-table] tr td:first-child"
```

Comment:
nit. Similar to the other files, can you assign a variable `const dataCyTableRows = "[data-cy=host-events-table] tr td:first-child"` and use it here? I think it will be nice for when we replace the antd! :)